commerce-theme-minium/commerce-theme-minium/src/css/minium/_variables.scss

```diff
@@ -44,14 +44,15 @@ $liferay-topbar-size-small: 60px;
 $frame-size: 70px;
 $frame-size-small: 56px;
 $sidebar-width: 300px;
+$liferay-sidebars-width: 320px;
 $content-max-width: 1440px;
 
 $transition: ease 200ms;
 $micro-transition: ease 100ms;
 
 $standard-radius: 8px;
 
-//breakpoints 
+//breakpoints
```

Comment:
🕺

Reply:
SF fun over comment intendedly left by Fabio since the previous PR on this file.